_posts/2022-04-24-sr2022-knockouts.md

```diff
@@ -0,0 +1,117 @@
+---
+title: TEAM 1st wins Student Robotics Competition 2022!
+image: image
+image_alt: Alt text for the image
+description: SR2022 came to an exciting conclusion with a victory for HRS
+---
+
+That’s it for Student Robotics 2022! HRS took the crown in an intense
+showdown between CCR, BRK, and QMC. It was fantastic to finally
+see robots in-person again after two years of being virtual. The ingenuity and
+engineering displayed by all our teams was incredible and we're delighted with
+the performances we saw over the weekend.
+
+## The Game: This Way Up
+
+This Way Up challenged teams to collect the most tin cans from the arena and
+ensure that they were the right way up for maximum points. When in a team’s
+scoring zone, correctly oriented cans were worth 3 points, upside down cans were
+worth one point, and cans lying on their side were worth nothing. However, not
+all cans were created equal: tin cans on the arena floor started upside down,
+but those on the raised platform in the centre were already the right way up,
+making them harder to get but easier to score points with. Robots could detect a
+can’s orientation by the coloured insulating band along the bottom. The states
+of the cans were recorded at the end of the matches and the teams were scored
+accordingly. {INSERT PICTURE OF ARENA}
+
+## League Matches
+
+Competition matches started with the league. Teams were awarded 8 points for
+coming first, 6 points for second, 4 points for third, and 2 points for fourth
+in each of their league matches to add to their league score. Teams played
+101 matches giving them time to improve their robot and tweak
+their strategy as the league went on. Teams had various designs from {list of
+different designs/strategies}. Particularly notable were {list a few teams that
+stood out to us}. {INCLUDE A FEW PICS OF THESE TEAMS}
+
+## Knockout Matches
+
+The league matches were important as they seeded the knockouts giving a
+potentially easier set of matches to teams placed higher in the league.
+{ANYTHING INTERESTING THAT HAPPENED}
+
+## The Final
```

Comment:
This section ends up including a bunch of stuff which isn't the final. Perhaps it would benefit from a different title ("Prizes" maybe?) or splitting out the other prizes into their own section?